### DISCUSSION TITLE
Livewire Overrules Alpine's x-cloak

### DISCUSSION THREAD

**Discussion by splashsky** [livewire/livewire#4771] [7. Bugs (archived)]
I've got a relatively simple project.

In my mobile-centric layout, I have a page that shows info for a given product. There are tabs to change what info is displayed. Each section is hidden by `x-cloak` and `x-show`, and managed by an AlpineJS component on the parent node of the sections. This works perfectly fine.

However, when I use a Livewire component at all - in this case, it's within the sections that Alpine manages - Livewire seems to ignore or overrule Alpine's `x-cloak`, and all content will flicker as it loads in. This only happens when using the Livewire component.

This has been the case any time I try to use Alpine and Livewire together, and I've been yet unable to find a fix.

**Comment by joshhanley** [comment]
@splashsky any chance you could create a demo of the issue on https://laravelplayground.com/ ?

Thanks!
